planetiler-custommap/src/main/resources/samples/shortbread.yml

```diff
@@ -610,6 +610,252 @@ layers:
+      shop: &poi_filter_shop
+        - alcohol
+        - bakery
+        - beauty
+        - beverages
+        - bicycle
+        - books
+        - butcher
+        - car
+        - chemist
+        - clothes
+        - computer
+        - convenience
+        - department_store
+        - doityourself
+        - dry_cleaning
+        - florist
+        - furniture
+        - garden_centre
+        - general
+        - gift
+        - greengrocer
+        - hairdresser
+        - hardware
+        - jewelry
+        - kiosk
+        - laundry
+        - mall
+        - mobile_phone
+        - newsagent
+        - optician
+        - outdoor
+        - shoes
+        - sports
+        - stationery
+        - supermarket
+        - toys
+        - travel_agency
+        - video
+      tourism: &poi_filter_tourism
+        - artwork
+        - alpine_hut
+        - bed_and_breakfast
+        - camp_site
+        - caravan_site
+        - chalet
+        - guest_house
+        - hostel
+        - hotel
+        - information
+        - motel
+        - picnic_site
+        - theme_park
+        - viewpoint
+        - zoo
+    attributes: &poi_attrs
+     # For the attributes below, we need to include_when, as otherwise
+     # the any-filter above could add variants if contains one matching and one non-matching variant
+    - key: amenity
+      include_when:
+        amenity: *poi_filter_amenity
+    - key: leisure
+      include_when:
+        leisure:  *poi_filter_leisure
+    - key: tourism
+      include_when:
+        tourism:  *poi_filter_tourism
+    - key: shop
+      include_when:
+        shop:  *poi_filter_shop
+    - key: man_made
+      include_when:
+        man_made:  *poi_filter_man_made
+    - key: historic
+      include_when:
+        historic:  *poi_filter_historic
+    - key: emergency
+      include_when:
+        emergency:  *poi_filter_emergency
+    - key: highway
+      include_when:
+        highway:  *poi_filter_highway
+    - key: office
+      include_when:
+        office:  *poi_filter_office
+    - *name
+    - *name_en
+    - *name_de
+    - key: housename
+      tag_value: addr:housename
+    - key: housenumber
+      tag_value: addr:housenumber
+    - key: cuisine
+      include_when:
+        amenity:
+          - restaurant
+          - fast_food
+          - pub
+          - bar
+          - cafe
+    - key: sport
+      include_when:
+        leisure:
+          - pitch
+          - sports_centre
+    - key: vending
+      include_when:
+        amenity: vending_machine
+    - key: information
+      include_when:
+        tourism: information
+    - key: tower:type
+      include_when:
+        man_made: tower
+    - key: religion
+      include_when:
+        amenity: place_of_worship
+    - key: denomination
+      include_when:
+        amenity: place_of_worship
+    - key: recycling:glass_bottles
+      type: boolean
+      value: '${ feature.tags["recycling:glass_bottles"] == "yes" }'
```

Comment:
I think this is the best way to express this relationship.
Unsure if there are better ways..

```py
if amenity=recycling:
   value = recycling:glass_bottles == yes
```

Reply:
Alternatively I think you can do:

```yaml
value:
  - if: { amenity: recycling, "recycling:glass_bottles": yes }
    value: true
```

dropping down to the CEL expressions is a slight performance hit compared to yaml-based expressions - although it's probably very minor when you limit evaluation with the include_when expression

Reply:
good point, did not think of combinding this in this way.
=> https://github.com/onthegomap/planetiler/pull/1207#discussion_r2003342519

We cannot remove the `include_when`, as above would only be
```py
if amenity == recycling and recycling:glass_bottles == yes
    return True
None